components/media-player/MediaPlayerOpen.vue

```diff
@@ -198,19 +214,19 @@ const {
           </button>
         </div>
       </div>
-      <ul class="px-3 pb-3">
-        <li v-for="(item, i) in queue" :key="i" @click="queue.index = i">
+      <ul ref="queueListElement" class="px-3 pb-3">
+        <li v-for="(item, i) in queue" :key="item.id" @click="queue.index = i">
           <div
             :class="
-              queue.index === i ? 'bg-tint text-black-1 hover:bg-tint' : ''
+              isCurrentTrack(i) ? 'bg-tint text-black-1 hover:bg-tint' : ''
             "
-            class="flex cursor-pointer justify-between gap-2 rounded-xl px-3 py-2 transition-all duration-500 ease-out hover:bg-background-2"
+            class="flex cursor-row-resize justify-between gap-2 rounded-xl px-3 py-2 transition-all duration-500 ease-out hover:bg-background-2"
```

Comment:
@sifferhans @SimonSimCity I think it makes sense to use a different mouse cursor to indicate that it's movable

Reply:
Yeah, good point!